package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "scripts": {
     "pretest": "eslint lib/ spec --fix",
-    "test": "mocha spec",
+    "test": "nyc mocha spec",
```

Comment:
What is the value of using `nyc`?

Reply:
It show test coverage and helps to find uncovered (untested) lines

Reply:
🤝